docs/schedule/process.zh.md

```diff
@@ -0,0 +1,336 @@
+
+**2011年9月26日**<br/>
+
+* 在规范中添加了'feed_info'建议：[讨论](https://groups.google.com/forum/#!topic/gtfs-changes/Sh0e4o9o2Gw)
+
+**2011年9月6日**<br/>
+
+* 在规范中添加了'agency_fare_url'建议：[讨论](https://groups.google.com/forum/#!topic/gtfs-changes/Zp9rPG07CgE)
+* 在规范中添加了 "exact_times "建议：[讨论](https://groups.google.com/forum/#!topic/gtfs-changes/nZF9lbQ7TQs)。
+
+**2009年3月30日**<br/>
+
+* 新增了一个关于公开提供交通信息的部分。这一点以前没有在小组里讨论过，因为严格来说，这并不是对数据的解释或编写方式的改变。然而，谷歌的一些人认为，包括对GTFS的非谷歌用途的讨论会很有意义，因为有越来越多的应用程序可以利用GTFS数据。
+* CSV格式的澄清：[讨论](https://groups.google.com/forum/#!topic/gtfs-changes/03qz5aTA2mk)。
+* 关于如何在route_color和route_text_color字段的描述中选择对比的颜色的额外指导。
+* 在这些主题中提出并测试的trip_short_name：a和b。
+* 修正了文件end的样本数据中的一个小错误（把S7站作为父站S8）。
+* 根据Marcy在评论期间的建议，在文件end的样本数据中添加了 "agency_lang "信息：[讨论](https://groups.google.com/forum/#!topic/gtfs-changes/5qP1kDUFqx0)。
+* 更新了侧边栏中OCTA的GTFSfeed的链接。
+* 见[原始摘要](https://groups.google.com/forum/#!topic/gtfs-changes/cL1E4oKKpKw)。
+
+**2009年2月26日**<br/>
+
+* 删除了大部分针对谷歌的feed提交说明，因为目前还有许多其他的应用程序在消费GTFS数据。
+* 修正了侧边栏中指向橙县OCTA公共馈送的断裂链接。
+
+**2008年8月7日**<br/>
+
+* 恢复了stop_url字段，该字段在8月6日的版本中被意外地省略了。
+* 在样本数据中增加了机构电话
+* 在向Google提交feed时，增加了对数据使用协议的提及。
+
+**2008年8月6日**<br/>
+
+* 增加了transfers.txt文件，允许feed发布者提供关于首选传输行为的提示[(原始建议](https://groups.google.com/forum/#!topic/gtfs-changes/cL1E4oKKpKw)）。
+* 在stops.txt中增加了location_type和parent_station字段，允许将停车点归入车站[原始提议](https://groups.google.com/forum/#!topic/gtfs-changes/ScGAyZ9a_yw)）。
+* 增加了agency_phone字段，用于提供一个机构的语音电话号码[原始提案](https://groups.google.com/forum/#!topic/gtfs-changes/8Itt58ueyqA)）。
+* 增加了 "测试你的报文 "一节，提到了开源的测试工具
+* 增加了对CSV格式、机构时区、机构语言、路线颜色、路线文本颜色、到达时间、出发时间、calendar.txt与calendar_dates.txt、票价表和frequencies.txt的说明。
+* 增加了链接到馈送历史文件，并更正了一些公共馈送链接
+* 更新了示例图片以描述当前的谷歌地图用户界面
+* 更新/修正了文件中的样本数据
+
+**2008年2月29日**<br/>
+
+* 在stops.txt中增加了 stop_code 字段，允许指定面向乘客的车站代码[原始提案](https://groups.google.com/forum/#!topic/gtfs-changes/k9A95fYZexc)）。
+* 澄清了routesroutes.txttxt中的route_short_name和route_long_name的描述。
+* 澄清了stop_times.txt中的 arrival_time 和 departure_time 的描述。
+* 修正了样本数据部分的错别字
+
+**2007年11月20日**<br/>
+
+* 澄清了block_id的描述
+* 修改了语言，不再强调Google Transit（因为非Google应用程序正在使用GTFS，而且交通路线选择现在是Google Maps的一个综合功能），并修正了各种错别字
+* 更新了截图示例，以反映GTFS字段在当前谷歌地图用户界面中的表现形式。
+* 更新了交通数据提供者的谷歌联系电子邮件地址
+* 更新了格式
+
+**2007年10月5日**<br/>
+
+* 修改了stop_sequence和shape_pt_sequence，允许任何增加的非负整数。
+* 澄清了描述并修正了错别字
+
+**2007年5月31日**<br/>
+
+* 更新了页面风格，使HTML更简洁，更容易访问
+* 增加了公共馈电实例和其他有用网站的链接
+* 删除了个别字段描述中的例子
+
+**2007年4月9日**<br/>
+
+* 增加了关于[提交信息源](https://developers.google.com/transit/google-transit#SubmitFeedToGoogle)的部分。
+* 增加了[示范交通机构馈送的例子](https://developers.google.com/transit/gtfs/examples/gtfs-feed)。
+* 增加了注意，如果所有的服务日期都在calendar_dates.txt中定义，可以省略calendar.txttxt。
+* 在只包含一个机构的馈送中，使agency_id字段成为可选项。这允许现有的feeds没有agency_id而保持有效。
+* 增加了agency_url、stop_url和route_url的更全面规范，以及这些字段的额外示例值。
+* 增加了6（Gondola）和7（Funicular）作为有效的route_type值。
+
+**2007年3月8日**<br/>
+
+* 稍作编辑，将stop_url字段从stop_times.txt（在2月28日的更新中，该字段被错误地指定为stop_times.stops.txt）移至stops.txt，即它所属的位置。
```

Comment:
something is wrong here, gtfs.org does not have brackets in this line, and there seems to be an extra stops.txt?